### PR TITLE
Handle unsupported targets gracefully

### PR DESCRIPTION
**Pull request by agraul**
The CPU-Mitigations formula is only supported on SLE12.3 and newer.
For other systems, this formula should be a no-op.

Without this commit, the following {% if supported %} check fails since
`supported' is not defined for unsupported systems.

References: bsc#1179273
Fixes: https://github.com/SUSE/spacewalk/issues/13268

### CODE REVIEW

**Comment by witekest** [comment]
Make sure to bump the version number in [Makefile](https://github.com/SUSE/salt-formulas/blob/master/Makefile). You can use it to build the tarball for OBS.
Please also note that we have `Manager-4.0` branch. Remember to cherry-pick your bugfix there if needed.

**Comment by agraul** [comment]
The Makefile version bump should be in this PR, right? Let me add that @witekest

File: cpu-mitigations-formula/states/init.sls [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 {% if grains['os_family'] == 'Suse' and grains['osrelease'] in supported_vers %}
```
Comment (witekest) [inline cpu-mitigations-formula/states/init.sls:8]:
```suggestion
{% set supported = (grains['os_family'] == 'Suse' and grains['osrelease'] in supported_vers) %}
```

Comment (agraul) [reply cpu-mitigations-formula/states/init.sls]: Right, that's a bit cleaner. Thanks!

**Review by witekest — APPROVED** [review]
Please remember about cherry-picking to stable branch if needed.

**Pull request merged**